.travis.mk

```diff
@@ -205,7 +205,7 @@ build_asan_debian:
 		-DENABLE_FUZZER=ON ${CMAKE_EXTRA_PARAMS}
 	make -j
 
-test_asan_debian_no_deps: build_asan_debian
+test_asan_debian_no_deps: build_asan_debian deps_debian
```

Comment:
It looks strange: `..._no_deps` depends on `..._deps`. At least should be clarified.

Reply:
It's not needed on ci, we don't run targets with dependencies installation there.
Hunk reverted.